hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/store/StateStoreService.java

```diff
@@ -116,7 +116,7 @@ public class StateStoreService extends CompositeService {
   /** Service to maintain State Store caches. */
   private StateStoreCacheUpdateService cacheUpdater;
   /** Time the cache was last successfully updated. */
-  private long cacheLastUpdateTime;
+  private long cacheLastUpdateTime = 0;
```

Comment:
Is this change necessary? The default value of long is `0`.

Reply:
Thanks for review.

Yes, the default value of long is 0, but  I think it is better to assign to 0 for emphasis on completing initialization.

Reply:
Thanks for the explanation! But the code looks weird because other variables are initialized in the constructor, can we initialize to `0` in the constructor? Just a personal opinion, let's wait goiri's view.

Reply:
Good idea, make it initialized in the constructor. Thanks.